packages/now-cli/src/commands/scale.js

```diff
@@ -239,7 +239,7 @@ export default async function main(ctx) {
     return 1;
   }
   if (deployment.version === 2) {
-    output.error('Cannot scale a deployment containing builds');
+    output.error('Cannot scale a V2 deployment');
```

Comment:
```suggestion
    output.error('Cannot scale a Now 2.0 deployment');
```
We try to use `Now 2.0` when possible instead of `v2` for messages.